zipkin-local.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+const _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+const {
+  THEME_OPTIONS,
+} = require("gatsby-plugin-newrelic-test/utils/constants");
+const { getCiData } = require("./utils/getCiData");
+const ciAttributes = getCiData();
+exports.__esModule = true;
+exports.stop = exports.create = void 0;
```

Comment:
What is the purpose of these two lines?

If we need those here, we should add some comments describing what the purpose of them is. Also, as a nitpick, can we group all our `require` statements at the top of this file and then lines like this after?

Reply:
Furthermore, we might want to put `exports` lines at the _bottom_ of this file (I've seen that convention used before).

Here's a common file layout I've seen:

```
import / require statements

constant definitions

function definitions

exports
```

Reply:
i'll have to double check the exact functionality there, these were part of the original `zipkin-local.js` file provided by Gatsby, which is generated by [this Typescript file](https://github.com/gatsbyjs/gatsby/blob/54d4721462b9303fed723fdcb15ac5d72e103778/packages/gatsby/src/utils/tracer/zipkin-local.ts). This seems to be added when it's [compiled from Typescript](https://2ality.com/2017/01/babel-esm-spec-mode.html) but i'd have to look into it further 🤔

but good point on the grouping